Zeeguu Reader/Zeeguu Reader/AddFeedTableViewController.swift

```diff
@@ -102,6 +113,7 @@ class AddFeedTableViewController: UITableViewController {
 	}
 	
 	func cancel(sender: UIBarButtonItem) {
+		urlField.resignFirstResponder()
 		self.dismissViewControllerAnimated(true, completion: { () -> Void in
 			self.delegate?.addFeedDidCancel()
```

Comment:
At the moment, it is an instance of `FeedOverviewTableViewController`, however, the constructor of `AddFeedTableViewController` expects a delegate, so it doesn't make sense to add a comment stating which class instance the delegate is, because, as far as this class (`AddFeedTableViewController`) is concerned, the delegate can be anything (as long as it conforms to the protocol).